doc/sphinx/proof-engine/ltac.rst

```diff
@@ -107,7 +107,7 @@ mode but it can also be used in toplevel definitions as shown below.
                      : | solve [ `expr` | ... | `expr` ]
                      : | idtac [ `message_token` ... `message_token`]
                      : | fail [`natural`] [`message_token` ... `message_token`]
-                     : | fresh | fresh `string` | fresh `qualid`
+                     : | fresh [ `component` | … | `component` ]
```

Comment:
I don't understand the use of the vertical bars. Shouldn't it be like for `idtac` above?

Reply:
Indeed… I can’t remember where the meta-syntax is documented…